Dockerfile.xcode

```diff
@@ -12,9 +14,8 @@ RUN dnf -y install --setopt=install_weak_deps=False \
     git clone https://github.com/NiklasRosenstein/pbzx --progress && \
     cd pbzx && \
     git checkout 2a4d7c3300c826d918def713a24d25c237c8ed53 && \
-    clang -O3 -llzma -lxar -I /usr/local/include pbzx.c -o pbzx
-
-CMD mkdir -p /root/xcode && \
+    clang -O3 -llzma -lxar -I /usr/local/include pbzx.c -o pbzx && \
+    mkdir -p /root/xcode && \
```

Comment:
CMD is json formatted. Rather than continue this separate RUN/CMD and putting in all those commands into json format, I wrapped it all up into RUN. We don't need separate docker layers for the two steps do we?